src/pages/UserGroups/UserGroups.tsx

```diff
@@ -0,0 +1,49 @@
+import { useApi } from "../../contexts/ApiProvider";
+import { useQuery } from "react-query";
+import { UserGroups } from "../../data/models/User";
+import Spinner from "../../components/Spinner";
+import GroupCard from "./GroupCard";
+import Button from "../../components/Button";
+
+export default function UserGroups() {
+  const { api } = useApi();
+  const { isLoading, data, isError } = useQuery<UserGroups>("user_groups", () =>
```

Comment:
Can you add the user groups query key to the constants file and follow the format I used for private entries? To keep things standardized to make sure we have ways to refresh all the data in the app.